pkg/config/config.go

```diff
@@ -26,15 +26,16 @@ type Config struct {
 	} `json:"server"`
 
 	SCIM struct {
-		CreateEmailIdentities bool            `json:"create_email_identities"`
-		CreateRoleGroups      bool            `json:"create_role_groups"`
-		GroupMappings         []ObjectMapping `json:"group_mappings"`
-		UserMappings          []ObjectMapping `json:"user_mappings"`
-		UserObjectType        string          `json:"user_object_type"`
-		GroupMemberRelation   string          `json:"group_member_relation"`
-		GroupObjectType       string          `json:"group_object_type"`
-		IdentityObjectType    string          `json:"identity_object_type"`
-		IdentityRelation      string          `json:"identity_relation"`
+		CreateEmailIdentities  bool            `json:"create_email_identities"`
+		CreateRoleGroups       bool            `json:"create_role_groups"`
+		GroupMappings          []ObjectMapping `json:"group_mappings"`
+		UserMappings           []ObjectMapping `json:"user_mappings"`
+		UserObjectType         string          `json:"user_object_type"`
+		GroupMemberRelation    string          `json:"group_member_relation"`
+		GroupObjectType        string          `json:"group_object_type"`
+		IdentityObjectType     string          `json:"identity_object_type"`
+		IdentityRelation       string          `json:"identity_relation"`
+		InvertIdentityRelation bool            `json:"invert_identity_relation"`
```

Comment:
Perhaps, instead of adding a boolean option, we can change the format of `IdentityRelation` to be `object#relation`. That way it can be set to either `user#identifier` or `identity#identifier`.